en/config_vtol/vtol_duo_tailsitter_config.md

```diff
@@ -0,0 +1,94 @@
+# Duo-Tailsitter VTOL Configuration
+
+This is the configuration tuning documentation for a "Duo-Tailsitter VTOL".
+
+A tailsitter is essentially a fixed wing vehicle that takes off in a pitched up configuration (essentially sitting on its tail, hence the name) and then transitions to fixed wing flight by pitch down by 90 degrees to achieve the fixed wing configuration.
+The Duo-Tailsitter variant is so-named because it has two motors (a "quad" tailsitter would have four).
+
+Like other VTOLs, the Duo-Tailsitter has two modes of operation: multicopter (MC) mode and fixed wing (FW) mode.
+In MC mode, lift and roll are dependent on the motors, while the pitch and yaw depend on the ailevons.
+In FW mode, yaw depends on differential thrust from the motors, while roll and pitch depend on the ailevons.
+
+For airframe specific documentation and build instructions see [VTOL Framebuilds](../frames_vtol/README.md).
+
+## Firmware & Basic Settings
+
+1. Run *QGroundControl*
+2. Flash the firmware for your current release or master (PX4 `main` branch build).
+3. In the [Frame setup](../config/airframe.md) section select the appropriate VTOL airframe.
+   At time of writing, this must be the [Generic VTOL Tailsitter](../airframes/airframe_reference.md#vtol-tailsitter).
+
```

Comment:
1. Should we recommend any other config before we start? I.e. the basic config for all frames?
2. Is there any autotune for Duo? If not, we should probably explicitly say so.

Reply:
1. IMO the basic config works only till basic PID tuning starts. After that its different.

2. I don't think there is autotune for Duo. Will mention that in the next commit.

Reply:
Thanks. So we should add
```suggestion
4. Perform all the other [standard configuration](../config/README.md), including compass, radio setup, etc., except for autotuning.

   :::note
   Autotuning is not supported for a duo tailsitter!
   :::
```